### PR TITLE
send noti when admin update order status

### PR DESCRIPTION
**Pull request by ariesh2t**
## Related Tickets:
- [#48098](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/48098)

## WHAT this PR does?
send notification to user when admin updates order status

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2022-04-13 10-21-58

### CODE REVIEW

**Comment by ariesh2t** [comment]
ready

File: app/Http/Controllers/AdminOrderController.php [hunk 1]
```diff
@@ -97,12 +104,43 @@ public function edit($id)
...
+                $data = [
+                    'order_id' => $order->id,
+                    'title' => 'title noti 2',
+                    'content' => 'content noti 2',
+                ];
+            } elseif ($request->order_status_id == config('orderstatus.delivered')) {
+                $data = [
+                    'order_id' => $order->id,
+                    'title' => 'title noti 3',
+                    'content' => 'content noti 3',
```
Comment (huyenttm-0901) [inline app/Http/Controllers/AdminOrderController.php:137]: data kiểu gì thế này Hoàng?

File: app/Http/Controllers/OrderController.php [hunk 2]
```diff
@@ -147,6 +135,13 @@ public function updatestatus($id)
                 $product->update();
             }
             $order->update();
+            $data = [
+                'order_id' => $order->id,
+                'title' => 'title noti 4',
+                'content' => 'content noti 4',
+            ];
```
Comment (huyenttm-0901) [inline app/Http/Controllers/OrderController.php:142]: đây nữa

**Comment by ariesh2t** [comment]
ready

**Pull request merged**